_sass/components/header.sass

```diff
@@ -7,10 +7,10 @@
 
     > .link > .selfie
         width: 125px
+        height: 125px
         margin-bottom: 25px
         border-radius: 100%
         transition: all .2s
-        box-shadow: 0
```

Comment:
Why that?

Reply:
This tag is redundant. This style modification appears in lines 13 and line 17 of this file.